### PR TITLE
added packager and cleaned package.son

### PR DESCRIPTION
**Pull request by aryue123321**
I have tested the windows package on windows 10.

### CODE REVIEW

**Comment by MatthewMerrill** [comment]
testing on linux

File: package.json [hunk 1]
```diff
@@ -1,28 +1,30 @@
 {
-  "name": "seniorproject",
+  "name": "savis",
+  "productName": "SAVIS",
   "version": "1.0.0",
-  "description": "1. setup jekyll",
+  "description": "Statistical Analysis Vizualizions for Introtuctory Statistics",
```
Comment (MatthewMerrill) [inline package.json:5]:
```suggestion
  "description": "Statistical Analysis Visualizations for Introductory Statistics",
```

File: package.json [hunk 2]
```diff
@@ -1,28 +1,30 @@
...
     "mocha": "^5.2.0",
-    "electron": "^4.1.4"
+    "puppeteer": "^1.14.0"
   },
   "scripts": {
     "test": "mocha --require babel-core/register",
     "test-all": "mocha --require babel-core/register --timeout=30000 test/**/*.test.js",
-    "start": "electron ."
+    "start": "electron .",
+    "package-mac": "electron-packager . --overwrite --platform=darwin --arch=x64 --prune=true --out=dist",
```
Comment (MatthewMerrill) [inline package.json:25]: I don't have a mac, can we test this on one? I can rent a laptop from the library tomorrow to test. Without knowing this library or having tried it yet, it seems strange that this one has no "savis" as an arg.

**Review by MatthewMerrill — CHANGES_REQUESTED** [review]

**Comment by MatthewMerrill** [comment]
works on my machine :+1:

File: package.json (hunk 2 above)
Comment (aryue123321) [reply package.json:25]: yeah, i noticed that too.  Copied those from somewhere and wanna see what is the output on mac first

**Pull request merged**